pending/dancer_core_send_as.pod

```diff
@@ -0,0 +1,125 @@
+=encoding utf8
+
+=head2 send_as BEER => 'Christmas Cheer'
```

Comment:
I wasn't expecting it to be rendered as "code". So something like `<h2>send_as BEER =&gt; 'Christmas Cheer'</h2>`, or:

<h2>send_as BEER =&gt; 'Christmas Cheer'</h2>